### PR TITLE
Implement leaderboard

### PR DESCRIPTION
**Pull request by Oleg-BLR**
fix leaderboard but do not display tournamentId in the table of leaderboards

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -33,6 +40,8 @@ const App = () => {
           <Route path="/" exact component={Tournaments} />
           {/* Route for the Players component at the /players path */}
           <Route path="/players" component={Players} />
+          {/* Route for the Players component at the /players path */}
```
Comment (PolinaTolkachova) [inline src/App.js:43]: Please update comments

File: src/components/Leaderboards.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from 'react'; // Import React library along with useEffect and useState hooks
+import axios from 'axios'; // Import Axios for making HTTP requests
+
+// Functional component for displaying a list of leaderboards in a table format
+const Leaderboards = () => {
+    // State to hold tournament data, initialized as an empty array
```
Comment (PolinaTolkachova) [inline src/components/Leaderboards.js:6]: Please remove or update comments: tournament -> leaderbord

File: src/App.js (hunk 1 above)
Comment (Oleg-BLR) [reply src/App.js:43]: done

File: src/components/Leaderboards.js (hunk 2 above)
Comment (Oleg-BLR) [reply src/components/Leaderboards.js:6]: done i hope all

**Review by Oleg-BLR — COMMENTED** [review]
done, I hope everything

**Pull request merged**